yarn.lock

```diff
@@ -2121,6 +2107,21 @@
     semver "^7.3.5"
     tsutils "^3.21.0"
 
+"@typescript-eslint/eslint-plugin@^5.2.0":
```

Comment:
Is the change of `yarn.lock` necessary?

Reply:
> Is the change of `yarn.lock` necessary?

Modifications to `yarn.lock` caused by just running `yarn install`. I think the previous lock dependency was not fixed properly